docs/data_dictionaries/index.rst

```diff
@@ -3,20 +3,38 @@
 Data Dictionaries
 =================
 
+Data Processed & Cleaned by PUDL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The PUDL data dictionary provides detailed metadata for core and output tables in the PUDL database,
+including field names, data types, and descriptions
+for each field by table.
+
 .. toctree::
-   :caption: Data Processed & Cleaned by PUDL
    :maxdepth: 1
+   :titlesonly:
 
    pudl_db
 
+Raw, Unprocessed Data
+^^^^^^^^^^^^^^^^^^^^^
+For data sources where substantial effort is required to publish raw databases (e.g., FERC Form 1), the dictionaries below provide key metadata on the raw data sources (e.g., table names and descriptions).
+detailing the table names, links to their corresponding database tables,
+raw file names, page numbers,
+data reporting frequency, and table descriptions.
```

Comment:
I would go into a little more detail here and say something like: 

"Certain raw datasets (e.g. FERC Form 1) require additional effort to process. We load these raw sources into SQLite databases before feeding them into the PUDL data pipeline. The dictionaries below provide key metadata on these raw sources including table name, table description, links to corresponding database tables, raw file names, page numbers, and data reporting frequency.

Also @zaneselvans what exactly does the "Pages" column mean in these FERC1 table metadata descriptions? https://catalystcoop-pudl.readthedocs.io/en/nightly/data_dictionaries/ferc1_db.html